pyproject.toml

```diff
@@ -0,0 +1,26 @@
+[tool.poetry]
+name = "carrot-mapper"
+version = "0.1.0"
+description = ""
+authors = ["John Smith <john@example.com>"]
```

Comment:
`pyproject.toml` requires an `authors` field, so happy for suggestions on who/email this should be

Reply:
I don't believe we have a carrot email, so it's probably either me or Phil? I'm happy either way.

If you can add multiple, then both might be worth doing - I guess just need to check who's ok with their email being public here.